tests/sqlparser_common.rs

```diff
@@ -11864,6 +11895,56 @@ fn parse_load_data() {
     );
 }
 
+#[test]
+fn test_load_extension() {
+    let dialects = all_dialects_where(|d| d.supports_load_extension());
+    let only_supports_load_data_dialects =
+        all_dialects_where(|d| !d.supports_load_extension() && d.supports_load_data());
+    let not_supports_load_dialects =
+        all_dialects_where(|d| !d.supports_load_data() && !d.supports_load_extension());
```

Comment:
given we're only looking to test the load_extension syntax, would it make sense to only keep the test scoped to only `all_dialects_where(|d| d.supports_load_extension())` in order to keep the test focused?
Otherwise its not clear if/why the load_data syntax is relevant here. Then for the `not_supports_load_dialects` I think we can skip verifying the parser error on other dialects, since by definition they don't support the syntax so its not super important

Reply:
Good idea! Have Done!